src/actions/ping.ts

```diff
@@ -17,38 +17,38 @@ export const pingAction = async (
 
   retryCount++
   const url = `https://${cid}.ipfs.${endpoint}`
-  console.log(`${isTTY
+  logger.info(`${isTTY
     ? `${colors.bold(`[${retryCount}]`)}: Requesting content at ${url}`
     : `[${retryCount}]`}: Requesting content at ${url}`,
   )
   try {
     const response = await fetch(url, { signal: AbortSignal.timeout(timeout), redirect: 'follow' })
     if (response.status === 504) {
       if (retries > 1) {
-        console.log(`🔄 Retrying in ${retryInterval / 1000} seconds...`)
+        logger.info(`🔄 Retrying in ${retryInterval / 1000} seconds...`)
```

Comment:
logger adds emojis so using it here would have two emojis at the same time

Reply:
Added another type of logger.
The idea is to have the same function always do the logging.